akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -719,7 +721,11 @@ abstract class ShardCoordinator(
             case Some(shards) =>
               log.debug("Graceful shutdown of region [{}] with shards [{}]", region, shards)
               gracefulShutdownInProgress += region
-              continueRebalance(shards.toSet)
+
+              // only for rebalance if we have at least one active region
+              if (calculateActiveRegions().size > 1)
+                continueRebalance(shards.toSet)
```

Comment:
This seems right to me.
One question is if we should have similar condition in `case RebalanceTick`? It's currently looking at `state.regions`, which is the stored state. Maybe it should also avoid rebalance in same way?